src/CentreonRemote/Domain/Service/ConfigurationWizard/LinkedPollerConfigurationService.php

```diff
@@ -216,9 +223,9 @@ private function setBrokerOutputOfPoller($pollerId, PollerServer $remote, $addit
                 WHERE `config_id` = :id");
             $statement->bindParam(':id', $configId, \PDO::PARAM_INT);
             $statement->execute();
-            $configGRoupId = $statement->fetchColumn('config_group_id') + 1;
+            $configGRoupId = $statement->fetchColumn(intval('config_group_id')) + 1;
```

Comment:
why cast to int ?

Reply:
got the error "int expected but string given", so I removed the parameter's function because it is not useful